{{cookiecutter.project_name}}/src/{{cookiecutter.project_name}}/settings.py

```diff
@@ -37,16 +37,19 @@
     'django.contrib.messages',
     'django.contrib.staticfiles',
     'rest_framework',
+    'django_prometheus',
```

Comment:
shouldn't this be `django-prometheus` ? I presume we're using https://pypi.org/project/django-prometheus/

Reply:
That is correct, if you look under the quickstart section for the docs you linked. It's strange that there is an underscore rather than a dash.